src/synopsys-action/diagnostics.ts

```diff
@@ -0,0 +1,33 @@
+import * as artifact from '@actions/artifact'
+import {UploadResponse} from '@actions/artifact/lib/internal/upload-response'
+import {getWorkSpaceDirectory} from '@actions/artifact/lib/internal/config-variables'
+import * as fs from 'fs'
+import * as inputs from './inputs'
+import {UploadOptions} from '@actions/artifact/lib/internal/upload-options'
+
+export async function uploadDiagnostics(): Promise<UploadResponse> {
+  const artifactClient = artifact.create()
+  const pwd = getWorkSpaceDirectory().concat('/.bridge/')
+  let files: string[] = []
+  files = getFiles(pwd, files)
+  const options: UploadOptions = {}
+  options.continueOnError = false
+  if (inputs.DIAGNOSTICS_RETENTION_DAYS) {
+    options.retentionDays = parseInt(inputs.DIAGNOSTICS_RETENTION_DAYS)
```

Comment:
What will happen, if user passes non-numeric value? Shall we check and handle it here.

Reply:
handled